R/zchunk_LA115.roofPV.R

```diff
@@ -14,12 +14,12 @@
 #' @importFrom tidyr gather spread
 #' @author YourInitials CurrentMonthName 2017
 #' @export
-module_energy_LA115.roofPV_DISABLED <- function(command, ...) {
+module_energy_LA115.roofPV <- function(command, ...) {
   if(command == driver.DECLARE_INPUTS) {
     return(c(FILE = "common/iso_GCAM_regID",
              FILE = "energy/A15.roofPV_curves",
              FILE = "energy/A15.roofPV_TechChange",
-             "L100.Pop_thous_ctry_Yh"))
+             FILE = "socioeconomics/L100.Pop_thous_ctry_Yh"))
```

Comment:
Hi @aarony2j - this isn't right. The data should either be available when you try to run the chunk, or we need to temporarily 'inject' it from the old data system. Looks like the latter. So, please move this file into the `inst/extdata/temp-data-inject` folder, and change the reference here.